.gitignore

```diff
@@ -1,6 +1,4 @@
 **/__pycache__/
 build/
 dist/
-**/*.egg-info
-src/archemist/core/optimisation/.vscode/c_cpp_properties.json
-tests/optimisation_test/optimisation_manager_backup.py
```

Comment:
Any extra files should be either kept external to the repo or kept locally and not pushed with any changes, that way you don't need to add them to .gitignore. Please remove these two lines